models/classes/Copier/ItemClassCopier.php

```diff
@@ -53,10 +53,9 @@ public function copy(
 
     private function assertInItemsRootClass(core_kernel_classes_Class $class): void
     {
-        if (
-            $class->getUri() !== TaoOntology::CLASS_URI_ITEM
-            && !$class->isSubClassOf($class->getClass(TaoOntology::CLASS_URI_ITEM))
-        ) {
+        $rootClass = $class->getClass(TaoOntology::CLASS_URI_ITEM);
+
+        if (!$class->equals($rootClass) && !$class->isSubClassOf($class->getClass(TaoOntology::CLASS_URI_ITEM))) {
```

Comment:
I think you can avoid fetching the same class twice here

```suggestion
        $rootClass = $class->getClass(TaoOntology::CLASS_URI_ITEM);

        if (!$class->equals($rootClass) && !$class->isSubClassOf($rootClass)) {
```

Reply:
Good point 🙂 
Forgot to change this, thanks!

Reply:
Done